src/pages/account/update-account.tsx

```diff
@@ -0,0 +1,248 @@
+import { GetServerSideProps } from "next";
+import React, { FunctionComponent } from "react";
+import { Button } from "../../components/Button";
+import { FormErrorSummary } from "../../components/ErrorSummary";
+import {
+  Form,
+  FormFieldset,
+  FormGroup,
+  FormLegendPageHeading,
+} from "../../components/Form";
+import { Grid } from "../../components/Grid";
+import { FormInputProps, Input } from "../../components/Input";
+import { Layout } from "../../components/Layout";
+import { IfYouNeedHelp } from "../../components/Mca";
+import { GovUKBody, SectionHeading } from "../../components/Typography";
+import { IAccountHolderDetails } from "../../entities/accountHolderDetails";
+import {
+  BeaconsGetServerSidePropsContext,
+  withContainer,
+} from "../../lib/container";
+import { FieldManager } from "../../lib/form/fieldManager";
+import { FormJSON, FormManager } from "../../lib/form/formManager";
+import { Validators } from "../../lib/form/validators";
+import { redirectUserTo } from "../../lib/redirectUserTo";
+import { PageURLs } from "../../lib/urls";
+import { diffObjValues } from "../../lib/utils";
+
+export interface UpdateAccountPageProps {
+  form: FormJSON;
+  accountHolderDetails: IAccountHolderDetails;
+}
+
+const getPageForm = ({
+  fullName,
+  telephoneNumber,
+  addressLine1,
+  addressLine2,
+  townOrCity,
+  county,
+  postcode,
+  email,
+}: any): FormManager => {
```

Comment:
```suggestion
const definePageForm = ({
  fullName,
  telephoneNumber,
  addressLine1,
  addressLine2,
  townOrCity,
  county,
  postcode,
  email,
}: FormSubmission): FormManager => {
```

To stick with the naming convention on other pages?